main.py

```diff
@@ -20,5 +20,6 @@ def movie_by_link(req: ResolverRequest):
             "id": movie.movieID,
             "title": movie.data['title'],
             "year": movie.data['year'],
-            "rating": movie.data['rating']
+            "rating": movie.data['rating'],
+            "coverUrl": movie.data["cover url"]
```

Comment:
Add a trailing comma so the next diff is nicer

```suggestion
            "coverUrl": movie.data["cover url"],
```